health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/DynamicSVGComponent.js

```
@@ -24,7 +24,7 @@ const DynamicSVG = ({ type,appType, data }) => {
   }, [url, data]);
 
   return (
-    <div
+    <div style={{ marginTop: "0.5rem", marginBottom: "0.5rem" }} // TODO @Nabeel @jagan @bhavya scan through the app we should have s3 urls added in app directly
       dangerouslySetInnerHTML={{ __html: svgContent }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)